### ISSUE TITLE
feature: sheudle for boosts

### ISSUE THREAD

**Issue by PaulFree14** [stom79/mastalab#9]
it would be really usefull if boost could be sheudled aswell. 
There are often a couple of toots I want to boost once I check mastodon, but I don't like it to much if they are boosted all at once. ...that's a bit spammy for others. 

Also it would allow one to boost into different timezones, and make the timelines also being alive when onself sleeps, but others are awake and happy to read your boosts.

**Comment by stom79** [comment]
Thanks for the suggestion. The difficulty is to find an easy way to schedule a boost.
Currently, simple click is used to boost with the authenticated account. Long click is used to pick up another account for boosting.
One solution would be to suggest if the boost needs to be scheduled before the boost itself. But, I will have to add a feature in settings to ask this question (default: disabled).

**Comment by jalcine** [comment]
Something like this could be added into a long-click modal of sorts. Using the long-press to open a modal window here could be compelling (makes it a launching pad for other features to be added in the future).

**Comment by deutrino** [comment]
I agree with @jalcine and I would LOVE to see this feature, as one thing I like to do is curate interesting content, and scheduling makes this easier for me and more fun for followers who don't get deluged with tons of boosts when I happen to be online, or miss the boosts because they weren't online when I happened to be.

**Comment by stom79** [comment]
I closed #114 and I will add some inputs here. I started a project. https://github.com/stom79/mastalab/projects

**Comment by LeeteqXV** [comment]
Great idea.

I think that this might also be useful as a tool to remember to toot about
something later.

To do that, just need an extra "select task" option that actually let you
type a draft toot right there in the wizard flow and schedule it. But THOSE
toots should be given a separate (customiseable) "category" because they
are NOT going to be automatically sent out at that date, but appear as
reminders to be finished/edited before either being rescheduled fore
later/more efiting or actually "toot now".

#featureRequest :-)

@LeeteqXV

**Comment by deutrino** [comment]
I would be happy to do some testing in my spare time as needed
